Tic-Tac-Toe-Python/Player.py

```diff
@@ -0,0 +1,18 @@
+from abc import ABC, abstractmethod
+
+
+class TicTacToePlayer(ABC):
```

Comment:
why did we make this as an abstract class ? this is currently only being used in TicTacToeUserPlayer

Reply:
There can be a feature to create a computer player in which the next_move function will be implemented differently.